manage.py

```diff
@@ -108,21 +105,23 @@ def seed(exclude_interventions=False, keep_unmentioned=False):
     add_static_organization()
     add_static_relationships()
     add_static_roles()
-    db_maintenance()
     db.session.commit()
 
-    # Always update interventions on development systems
-    if app.config["SYSTEM_TYPE"].lower() == 'development':
-        exclude_interventions = False
-
     # import site export file if found
-    SitePersistence().import_(exclude_interventions, keep_unmentioned)
+    SitePersistence().import_(keep_unmentioned=keep_unmentioned)
 
 
-@app.cli.command()
-def export_site():
+@click.option('--dir', '-d', default=None, help="Export directory")
```

Comment:
Are you expecting we'll use manage.py to update orgs outside the release process (out of band)? If so, where will these changes be kept?